### PR TITLE
Bug 1160365 - [Navigation] Mobile user settings were in wrong order

### PR DESCRIPTION
**Pull request by fahimnadif69**
[Navigation] order is now set in proper order

### CODE REVIEW

File: src/templates/nav.html [hunk 1]
```diff
@@ -36,7 +35,8 @@
     {{ nav_item_settings(url('settings'), _('Settings'), 'settings',
                          'nav--settings--logged-in') }}
     {{ nav_item_settings(url('purchases'), _('My Apps'), 'purchases',
-                         'nav--settings--logged-in') }}
+                         'nav--settings--logged-in') }
```
Comment (atiqueahmedziad) [inline src/templates/nav.html]: You have deleted one `}` here. Don't need to delete this

**Comment by fahimnadif69** [comment]
Okay. I will try my best to fix it.

**Comment by mstriemer** [comment]
@fahimnadif69 Looks good! In the future please squash your commits together so we only have one commit in the pull request. You can do this by either amending your initial commit when you make an update or using rebase.

**Comment by mstriemer** [comment]
Also your commit message should describe the state this will be in after applying your patch. So `Correct ordering of user settings in menu (bug 1160365)` would be better.

Thanks for the patch!

**Pull request closed without merging**